### PR TITLE
My finished submission

### PR DESCRIPTION
**Pull request by LAakerberg**
Hi, I would like to have some feedback on my finished submission.

This project is based on the social media app we should create and is my own profile that's show the latest posts on my wall and some of my contacts.

So far I have only got one peer feedback and I made some changes about how last posted was showing up in the first page.
Changed from column to row instead. This changes made the site look more personally with unique posts instead of looking like a blog.

I have added some few extra of my own classes & scss style because finding the correct on in bootstrap/sass.

### CODE REVIEW

File: login.html [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+      </div>
+
+      <!-- Login form -->
+      <div class="px-2">
+        <div
+          class="w-80 border border-dark container-sm form-width bg-lightred rounded-2"
+        >
+          <div class="p-2">
+            <div class="mb-3">
+              <form>
```
Comment (jaaneh) [inline login.html]: You are missing the form `action` attribute here. The form should take you to /profile.html once a user signs in.

File: login.html [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+                </div>
+                <div class="form-floating py-1">
+                  <input
+                    type="password"
+                    class="form-control"
+                    id="floatingPassword"
+                    placeholder="Password"
+                  />
+                  <label for="floatingPassword">Password</label>
+                </div>
```
Comment (jaaneh) [inline login.html]: Your form inputs should include some client-side validation. Such as the `required` and `pattern` attributes.

File: index.html [hunk 3]
```diff
@@ -4,10 +4,197 @@
...
+              </li>
+            </ul>
+            <form role="search">
+              <input
+                class="form-control"
+                type="search"
+                placeholder="Search"
+                aria-label="Search"
+              />
+            </form>
```
Comment (jaaneh) [inline index.html:53]: Would be great to have a submit button, for example with a magnifying glass icon on it. Unless you're wanting results to show up as a dropdown for matching elements, such as with a typeahead search?

**Review by jaaneh — CHANGES_REQUESTED** [review]

File: login.html (hunk 2 above)
Comment (LAakerberg) [reply login.html]: Added required attributes on the login page.

File: index.html (hunk 3 above)
Comment (LAakerberg) [reply index.html:53]: Thanks for the tips. An dropdown search would have been nice actually. But for now I just added an search button with an magnifying glass icon!

**Pull request merged**